modules/extensions/src/fill-style/fill-style-extension.ts

```diff
@@ -5,21 +5,24 @@ import {patternShaders} from './shader-module';
 import type {
   Layer,
   LayerContext,
+  DefaultProps,
   Accessor,
   AccessorFunction,
   TextureSource,
   UpdateParameters
 } from '@deck.gl/core';
 import type {Texture} from '@luma.gl/core';
 
-const defaultProps = {
+const defaultProps: DefaultProps<FillStyleExtensionProps> = {
   fillPatternEnabled: true,
   fillPatternAtlas: {
     type: 'image',
     value: null,
     async: true,
     parameters: {
-      minFilter: 'linear'
+      // Override default mipmap filter 'linear', i.e. set MIN_FILTER to LINEAR instead of LINEAR_MIPMAP_LINEAR
+      // @ts-expect-error invalid value for type `SamplerProps` - luma.gl should allow unset
+      mipmapFilter: ''
```

Comment:
I wonder if it would be better to explicitly support `mipmapFilter: 'none'` in luma to make it clearer what is happening. None of the other `SamplerProps` allow falsy values

Reply:
I think `lodMaxClamp: 0` would have the effect we want? Luma is mirroring the WebGPU spec, in which all samplers have `mipmapFilter`, but you can restrict the range of mipmaps used with `lodMinClamp` / `lodMaxClamp`. 

Opened PR:

- https://github.com/visgl/deck.gl/pull/8677